pom.xml

```diff
@@ -17,8 +17,8 @@
 		<project.fullVersion>${project.version}</project.fullVersion>
 
 		<junit.version>5.8.2</junit.version>
-		<mockito.version>4.3.1</mockito.version>
-		<netty.version>4.1.74.Final</netty.version>
+		<mockito.version>4.4.0</mockito.version>
+		<netty.version>4.1.75.Final</netty.version>
```

Comment:
fwiw, the netty version spigot uses is [4.1.68](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/pom.xml#106) (so close)

Reply:
NachoSpigot is planning to bump Netty to 4.1.75

Reply:
It makes no difference here as that netty version is only used in tests (minecraft doesn't bundle netty anymore) 🤔